yamllint/conf/relaxed.yaml

```diff
@@ -6,9 +6,11 @@ rules:
   braces:
     level: warning
     max-spaces-inside: 1
+    max-spaces-inside-empty: -1
```

Comment:
This was a remnant from before I added backward compatibility (setting the *-empty values to their non-empty equivalents with -1). I removed it.